app/helpers/settings_rbac_tag_helper.rb

```diff
@@ -0,0 +1,32 @@
+module SettingsRbacTagHelper
+  private
+
+  def row_data_with_icon(label, value, icon)
+    {:cells => {:label => label, :value => value, :icon => icon}}
+  end
+
+  def settings_rbac_tag_summary
+    if session[:assigned_filters].empty?
+      miq_structured_list(
+        :title => "#{current_tenant.name} #{_("Tags")}",
+        :mode  => "settings_rbac_tag_details",
+        :rows  => [{:cells => {:value => _("No %{tenant_name} Tags have been assigned") % {:tenant_name => current_tenant.name}, :icon => "fa fa-tag", :style => "display_flex"}}]
+      )
+    else
+      rows = []
+      session[:assigned_filters].keys.sort.each do |k|
+        v = session[:assigned_filters][k]
+        value = ""
+        v.sort.each_with_index do |t, idx|
+          value += idx > 0 ? "| #{t}" : t
+        end
```

Comment:
Can you try this-

`values = session[:assigned_filters][k].sort.join(' | ')`

Reply:
implemented the changes